docs/design/job-resource-reservation-design.md

```diff
@@ -81,16 +82,67 @@ task type is unpredictable and complex.
 
 ![Feature Design](./images/reservation_design.png)
 
-## Implementation
+## Implementation(v1.1.0)
+Volcano v1.1.0 has implemented that recognise target job and reserve resource automatically.
 ### Action
-Add two new action: elect and reserve. Elect action aims to find the Big Job. Reserve action is responsible for select
+Add two new action: elect and reserve. Elect action aims to find the target job. Reserve action is responsible for select
 locked nodes.
+#### Elect
+If no target job is elected, select one from jobs whose podgroup is `pending` and satisfy conditions  in `TargetJob`
+function registered in session object.
+#### Reserve
+If target job exists and is not ready, reserve nodes for it according algorithm implemented in `ReservedNodes` function
+registered in session object.
 ### Plugin 
-Add new Plugin reservation to implement algorithm detail above.
-
+Add new Plugin reservation to implement algorithm detail about selecting target job and reserving nodes. `targetJobFn`
+selects job whose priority is the highest and waits for the longest time. `reservedNodesFn` reserve node whose idle resource
+is the max in every session.
 ![Workflow](./images/reservation_workflow.png)
+### Recommend practice
+An example how to make use of this feature is to configure scheduler's configuration as follows:
+```yaml
+actions: "enqueue, elect, allocate, backfill, reserve"
```

Comment:
why do we need `elect` action? A callback/plugin function in `reserve` should be enough.

Reply:
`elect` action is used to elect jobs labeled by user manually or selected by plugin automatically, then `reserve` action reserves nodes according to jobs and related plugins. So in my opinion, `elect` is necessary, but `reserve` could merge into `elect`, or filter nodes directly in `ssn.predicateFn()` by callback functions in `reservation` plugin.